build.gradle

```diff
@@ -158,18 +177,17 @@ subprojects {
 
   artifacts {
     archives srcJar
+    archives testJar
```

Comment:
Adding these to all subprojects doesn't currently work because some won't generate jars if they have nothing in these directories, e.g. `examples/src/` doesn't have a `test/` subdirectory, so the test jars won't be generated. In turn, anything depending on it (such as `uploadArchivesAll`, or more specifically `:examples:uploadArchives`) will fail.

(I think `uploadArchivesAll` should catch everything important, but you might spot-check a bunch of other command from the README too just to make sure they will continue to work as expected.)